samples/client/petstore/java/okhttp-gson/src/test/java/org/openapitools/client/ApiClientTest.java

```diff
@@ -335,7 +335,7 @@ public void testInterceptorCleanupWithNewClient() {
         OkHttpClient oldClient = apiClient.getHttpClient();
         assertEquals(1, oldClient.networkInterceptors().size());
 
-        OkHttpClient newClient = new OkHttpClient();
+        OkHttpClient newClient = apiClient.getHttpClient().newBuilder().build();
```

Comment:
CircleCI reports that this change gets overridden when the samples are updated:

```diff
diff --git a/samples/client/petstore/java/okhttp-gson/src/test/java/org/openapitools/client/ApiClientTest.java b/samples/client/petstore/java/okhttp-gson/src/test/java/org/openapitools/client/ApiClientTest.java
index 3675d8645a..6181e8afa2 100644
--- a/samples/client/petstore/java/okhttp-gson/src/test/java/org/openapitools/client/ApiClientTest.java
+++ b/samples/client/petstore/java/okhttp-gson/src/test/java/org/openapitools/client/ApiClientTest.java
@@ -335,7 +335,7 @@ public class ApiClientTest {
         OkHttpClient oldClient = apiClient.getHttpClient();
         assertEquals(1, oldClient.networkInterceptors().size());
 
-        OkHttpClient newClient = apiClient.getHttpClient().newBuilder().build();
+        OkHttpClient newClient = new OkHttpClient();
         apiClient.setHttpClient(newClient);
         assertEquals(1, apiClient.getHttpClient().networkInterceptors().size());
         apiClient.setHttpClient(newClient);
```

This is because the source for this file is in the `CI/samples.ci` folder. Can you update this file:
https://github.com/OpenAPITools/openapi-generator/blob/6824bd8b2445e54c25f26589ac51eac3de62f0e9/CI/samples.ci/client/petstore/java/test-manual/okhttp-gson/ApiClientTest.java#L338